records/config.go

```diff
@@ -23,6 +23,9 @@ type Config struct {
 	// Zookeeper: a single Zk url
 	Zk string
 
+	// Zookeeper Timeout: how long in seconds to wait for Zookeeper to be initially responsive (default 30)
+	ZkTimeout int
```

Comment:
Can we be more descriptive about which timeout this is configuring? `ZkConnectionTimeout`?